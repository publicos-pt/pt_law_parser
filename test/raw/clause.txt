Decreta que:
Título I
Princípios
I
PrincípiosI
Bla bla bla
