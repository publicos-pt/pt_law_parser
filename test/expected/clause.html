<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>
<p>Decreta que:</p>
<div class="title"><h3 class="title">Título I
<p>Princípios</p></h3>
<div class="clause"><h5 class="title">I
<p>PrincípiosI</p></h5>
<p>Bla bla bla</p></div></div></html>